src/managers/AdManager/AdManager.ts

```diff
@@ -21,82 +26,89 @@ class AdManager {
 
   private page: TPages | null;
   private productsIds: ReturnType<typeof getProductsIds> | [];
-  private websiteId: number;
 
   public injectAdnPixelScript = () => {
     const area = this.page ? this.mapPageToArea(this.page) : null;
 
     if (window.dlApi && dlApi.fetchNativeAd) {
-      dlApi.area = area || '';
-      dlApi.keyvalues = {
-        offer_ids: this.productsIds.toString(),
-        website_id: this.websiteId,
-      };
+      dlApi.area = area || 'ros';
+      dlApi.addKeyValue('offer_ids', this.productsIds.toString());
```

Comment:
Ta wartość powinna być przekazana do funkcji `dlApi.fetchNativeAd` jako argument `options.opts.offer_ids` (nie ma potrzeby rzutowania na string). W przeciwnym razie mogłoby się zdarzyć, że wszystkie pozostałe reklamy w ramach RMN byłyby zawsze dopasowane kontekstowo do podanych ofert, a takie zachowanie nie zawsze będzie korzystne.

Poza tym jeśli to już by miało być w tym miejscu, to powinno wyglądać tak:
```js
(dlApi.cmd = dlApi.cmd || []).push(function() {
    dlApi.addKeyValue('offer_ids', this.productsIds.toString());
});
```
W przeciwnym razie w tej linijce będzie *race condition* - czasem będzie działać poprawnie, a innym razem dostaniemy błąd:
> TypeError: dlApi.addKeyValue is not a function